addons/binding/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/util/Parse.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.opensprinkler.internal.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * The {@link Parse} class contains static methods for parsing JSON
+ * output based on key names.
+ *
+ * @author Chris Graham - Initial contribution
+ */
+public class Parse {
+    /**
+     * Parses an integer from a JSON string given its key name.
+     *
+     * @param jsonData The JSON formatted string to parse from.
+     * @param keyName The name of the object data to return.
+     * @return int value of the objects data.
+     */
+    public static int jsonInt(String jsonData, String keyName) {
+        JsonElement jelement = new JsonParser().parse(jsonData);
```

Comment:
I would think that it is not very efficient to instantiate a new JsonParser on every single call. Better create a field and reuse a single instance.